pyatlan/client/user.py

```diff
@@ -211,7 +211,7 @@ def get_all(
         :returns: a list of all the users in Atlan
         """
         response: UserResponse = self.get(offset=offset, limit=limit, sort=sort)
-        return [user for user in response]
+        return response
 
     @validate_arguments
     def get_by_email(
```

Comment:
Let's update this method as well, since it also returns a list of `AtlanUser`. We should have it return a `UserResponse` instead, which includes support for iterative pagination.

Reply:
Same for `get_by_emails()` method 👇